client/src/components/LayerManager.vue

```diff
@@ -128,7 +128,7 @@ export default defineComponent({
               confidencePairs: track.confidencePairs,
             };
             frameData.push(trackFrame);
-            if (frameData[frameData.length - 1].selected && (editingTrack)) {
+            if (trackFrame.selected && (editingTrack)) {
```

Comment:
this line was super weird.  I think `frameData[frameData.length - 1].selected` is always the same as `trackFrame.selected` given the push on line 130...